src/table.js

```diff
@@ -565,9 +569,9 @@ export function __table(source, operations) {
     }
   }
   for (const {column, direction} of reverse(operations.sort)) {
-    const compare = direction === "desc" ? descending : ascending;
+    const compare = (a, b) => ascendingDefined(a[column], b[column]);
     if (source === input) source = source.slice(); // defensive copy
-    source.sort((a, b) => compare(a[column], b[column]));
+    source.sort(direction === "desc" ? (a, b) => compare(b, a) : compare);
```

Comment:
Tiny optimization to avoid an additional wrapper function in the descending case by taking the same approach for both:

```suggestion
    source.sort(direction === "desc" ? (a, b) => ascendingDefined(b[column], a[column]) : (a, b) => ascendingDefined(a[column], b[column]));
```